build_scripts/windows/scripts/az.completion.psm1

```diff
@@ -0,0 +1,18 @@
+Register-ArgumentCompleter -Native -CommandName az -ScriptBlock {
+    param($commandName, $wordToComplete, $cursorPosition)
+    $completion_file = New-TemporaryFile
+    $env:ARGCOMPLETE_USE_TEMPFILES = 1
+    $env:_ARGCOMPLETE_STDOUT_FILENAME = $completion_file
+    $env:COMP_LINE = $wordToComplete
+    $env:COMP_POINT = $cursorPosition
+    $env:_ARGCOMPLETE = 1
+    $env:_ARGCOMPLETE_SUPPRESS_SPACE = 0
+    $env:_ARGCOMPLETE_IFS = "`n"
+    az 2>&1 | Out-Null
+    
+    Get-Content $completion_file | ForEach-Object {
+        [System.Management.Automation.CompletionResult]::new($_, $_, 'ParameterValue', $_)
+    }
```

Comment:
Could you provide an example of the content of this file?

Reply:
If the input is `az account `, this file will be:
```
-h
--help
list
set
show
clear
list-locations
get-access-token
lock
management-group
subscription
tenant
alias
accept-ownership-status
```